README.md

```diff
@@ -121,6 +121,7 @@ The following commands are available:
 * `doil delete <instance_name>` deletes an instance you do not need anymore
 * `doil status` lists the current running doil instances
 * `doil exec <instance_name> <cmd>` executes a bash command inside the instance
+* `doil sut` (alias for `doil instances:set-update-token`) Sets an update token as an environment variable. This is cross-checked during instance updates via the browser
```

Comment:
Bitte in der README die Funktionalität ein bisschen beschreiben, so dass sie auch ohne Geheimwissen eingesetzt werden kann.